docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,305 @@
+* Various model dependency updates @mglowacki100 (#3373)
+* Various code cleanup and logging improvements @Innixma (#3408, #3570, #3652, #3734)
+
+### Bug Fixes / Code and Doc Improvements
+* Fixed incorrect model memory usage calculation @Innixma (#3591)
+* Fixed `infer_limit` being used incorrectly when bagging @Innixma (#3467)
+* Fixed rare edge-case FastAI model crash @Innixma (#3416)
+* Various minor bug fixes @Innixma (#3418, #3480)
+
+## AutoMM
+[AutoGluon MultiModal (AutoMM)](https://auto.gluon.ai/stable/tutorials/multimodal/index.html) is designed to simplify the fine-tuning of foundation models for downstream applications with just three lines of code. 
+It seamlessly integrates with popular model zoos such as [HuggingFace Transformers](https://github.com/huggingface/transformers), 
+[TIMM](https://github.com/huggingface/pytorch-image-models), and [MMDetection](https://github.com/open-mmlab/mmdetection), 
+providing support for a diverse range of data modalities, 
+including image, text, tabular, and document data, whether used individually or in combination.
+
+### New Features
+
+* Semantic Segmentation
+    * Introducing the new problem type `semantic_segmentation`, 
+      for fine-tuning [Segment Anything Model (SAM)](https://segment-anything.com/) with three lines of code. @Harry-zzh @zhiqiangdon  (#3645, #3677, #3697, #3711, #3722, #3728) 
+    * Added comprehensive benchmarks from diverse domains, 
+      including natural images, agriculture, remote sensing, and healthcare.
+    * Utilizing parameter-efficient finetuning (PEFT) [LoRA](https://arxiv.org/abs/2106.09685), showcasing consistent superior performance over alternatives ([VPT](https://arxiv.org/abs/2203.12119), 
+      [adaptor](https://arxiv.org/abs/1902.00751), [BitFit](https://arxiv.org/abs/2106.10199),
+      [SAM-adaptor](https://arxiv.org/abs/2304.09148), and [LST](https://arxiv.org/abs/2206.06522)) in the extensive benchmarks.
+    * Added one [semantic segmentation tutorial](https://auto.gluon.ai/stable/tutorials/multimodal/image_segmentation/beginner_semantic_seg.html) @zhiqiangdon (#3716).
+    * Using [SAM-ViT Huge](https://huggingface.co/facebook/sam-vit-huge) by default (GPU memory > 25GB required).
+* Few Shot Classification
+    * Added the new `few_shot_classification` problem type for training few shot classifiers on images or texts. @zhiqiangdon (#3662, #3681, #3695)
+    * Leveraging image/text foundation models to extract features and train SVM classifiers.
+    * Added one [few shot classification tutorial](https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/few_shot_learning.html). @zhiqiangdon (#3662)
+* Supported [torch.compile](https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html) for faster training (experimental and torch >=2.2 required) @zhiqiangdon (#3520).
+
+### Performance Improvements
+
+* Improved default image backbones, achieving a 100% win-rate on the image benchmark. @taoyang1122 (#3738)
```

Comment:
@taoyang1122, could you please provide the benchmark config link here? Additionally, please specify the version (assuming version 0.8.2) that we have compared against.

Reply:
Yes, it is in comparison with 0.8.2. The config is just the default config.